pyneuroml/utils/CellBuilder.py

```diff
@@ -0,0 +1,353 @@
+            neuro_lex_id = neuro_lex_ids["soma"]
+            if use_convention:
+                seg_group_default = cell.get_segment_group("soma_group")
+        if "dend_" in group:
+            neuro_lex_id = neuro_lex_ids["dend"]
+            if use_convention:
+                seg_group_default = cell.get_segment_group("dendrite_group")
+
+        if seg_group is None:
+            seg_group = SegmentGroup(id=group, neuro_lex_id=neuro_lex_id)
+            cell.morphology.segment_groups.append(seg_group)
+
+        seg_group.members.append(Member(segments=segment.id))
+        # Ideally, these higher level segment groups should just include other
+        # segment groups using Include, which would result in smaller NML
+        # files. However, because these default segment groups are defined
+        # first, they are printed in the NML file before the new segments and their
+        # groups. The jnml validator does not like this.
+        # TODO: clarify if the order of definition is important, or if the jnml
+        # validator needs to be updated to manage this use case.
+        if use_convention and seg_group_default:
+            seg_group_default.members.append(Member(segments=segment.id))
+
+    if use_convention:
+        seg_group_all = cell.get_segment_group("all")
+        seg_group_all.members.append(Member(segments=segment.id))
+
+    cell.morphology.segments.append(segment)
+    return segment
+
+
+def set_init_memb_potential(cell: Cell, v: str, group: str = "all") -> None:
+    """Set the initial membrane potential of the cell.
+
+    :param cell: cell to modify
+    :type cell: Cell
+    :param v: value to set for membrane potential with units
+    :type v: str
+    :param group: id of segment group to modify
+    :type group: str
+    """
+    cell.biophysical_properties.membrane_properties.init_memb_potentials.append(
+        InitMembPotential(value=v, segment_groups=group)
+    )
+
+
+def set_resistivity(cell: Cell, resistivity: str, group: str = "all") -> None:
+    """Set the resistivity of the cell
+
+    :param cell: cell to modfify
+    :param resistivity: value resistivity to set with units
+    :type resistivity: str
+    :param group: segment group to modify
+    :type group: str
+    """
+    cell.biophysical_properties.intracellular_properties.resistivities.append(
+        Resistivity(value=resistivity, segment_groups=group)
+    )
+
+
+def set_specific_capacitance(cell: Cell, spec_cap: str, group: str = "all") -> None:
+    """Set the specific capacitance for the cell.
+
+    :param cell: cell to set specific capacitance for
+    :type cell: Cell
+    :param spec_cap: value of specific capacitance with units
+    :type spec_cap: str
+    :param group: segment group to modify
+    :type group: str
+    """
+    cell.biophysical_properties.membrane_properties.specific_capacitances.append(
+        SpecificCapacitance(value=spec_cap, segment_groups=group)
+    )
+
+
+def add_channel_density_v(
```

Comment:
How does this look @anujanegi ?

Is this fine, or do you think specific helper functions for each type of ChennelDensity.. would be better?

Note that this uses the `add()` function for MembraneProperties. You'll need to use the libNeuroML from this PR for that:

https://github.com/NeuralEnsemble/libNeuroML/pull/127

(the schema had a bug that we corrected, and so libNeuroML had to be regenerated).